.env

```diff
@@ -13,6 +13,7 @@ ODOO_PROXY_MODE=true
 # Domains
 DOMAIN_TEST=test.example.com
 DOMAIN_PROD=www.example.com
+DOMAIN_PROD_ALT=example.com
```

Comment:
Please could you add a comment? Like:

# You can add as many ALT domains as you want adding DOMAIN_PROD_ALT_2=subdomain.example.com, DOMAIN_PROD_ALT_3=othersubdomain.example.com, ...

Reply:
I only add comments with basic instructions in the scaffolding. Advanced instructions go in the project's README. I can add it there if you want, although it's pretty obvious IMHO :roll_eyes: